packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -154,7 +154,7 @@ pc.setContentSlides = (screenPc) => {
  * @returns {RTCPeerConnection}
```

Comment:
Let's update this to be more accurate:
```suggestion
 * @returns {Promise.<RTCPeerConnection>}
```